backend/audit/models.py

```diff
@@ -475,12 +475,22 @@ class Meta:
         ]
 
 
+def excel_file_path(instance, filename):
+    """
+    We want the actual filename in the filesystem to be unique and determined
+    by report_id and form_section--not the user-provided filename.
+    """
+    base_path = "excel"
+    report_id = SingleAuditChecklist.objects.get(id=instance.sac.id).report_id
```

Comment:
**question (non-blocking)**: Isn't this just `instance.sac.report_id`? If there is an `instance.sac` object to look up `id` on, then won't it also have a `report_id` attribute that we can use directly?

Reply:
No, no, it's better to throw in a redundant query just to make sure the oil flows smoothly in the database.